pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>Karedi</groupId>
 	<artifactId>Karedi</artifactId>
-	<version>1.6.0</version>
+	<version>1.6.0-ui-styling</version>
```

Comment:
Please leave the version as 1.6.0, I update it in master only on new version release.